src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -248,9 +249,7 @@ def __getattr__(self, key: str) -> Any:
                 if hasattr(native_index, key):
                     # Any methods that not supported by the current Index.py but exist in a
                     # native pandas index object should raise a not implemented error for now.
-                    raise ErrorMessage.not_implemented(
-                        f"Index.{key} is not yet implemented"
-                    )
+                    ErrorMessage.not_implemented(f"Index.{key} is not yet implemented")
```

Comment:
Is `raise ErrorMessage.not_implemented()` equivalent to just `ErrorMessage.not_implemented()`, or was this a bug?

Reply:
Just calling `ErrorMessage.not_implemented()` should handle the raising.

Reply:
But it used to have `raise` before it. So I'm suspecting this was a bug that was not captured by tests.

Reply:
It was kind like a bug because `ErrorMessage.not_implemented` raise internally.